### ISSUE TITLE
stdint.h

### ISSUE THREAD

**Issue by Hadatko** [fox-it/dissect.cstruct_legacy#4]
Hi,
Thanks for this project. Looks good for our purpose.
Would be great to support types from stdint.h like uint8_t, int8_t, ... by default without adding them manually, or support parsing this header file.

**Comment by Hadatko** [comment]
I just found this #3 . I understand the solution, but this is impact every embedded developer and for simplifying parsing process would be great to somehow simplified usage of these basic standard data types.

**Comment by Schamper** [comment]
I agree. We're in the process of reviewing a substantial refactor and plan to publish it soon. I don't think these types are in there right now, but I'll add them so that they make it into the next release.

**Comment by Hadatko** [comment]
Thank you very much

**Comment by Hadatko** [comment]
Do you have a release date for next release? We need to know for making decission for releasing our app.

**Comment by Schamper** [comment]
No release date yet. In the meantime, I can review, merge your changes and publish a new version. I can do this when I'm back at work after new years.

**Comment by Hadatko** [comment]
This is great. Thank you. Maybe you will find some thoughts for new version. 

i wanted support also this type of declaration (struct inside struct):
```
struct A{
   struct {
     int32_t a;
  } param1;
};
```
But with current parser it is harder to do it (to support all cases). So menatime i am fine with alternative:
```
struct B{
     int32_t a;
  };
struct A{
    struct B param1;
};
```
Enjoy your PTO ;)

**Comment by Schamper** [comment]
Yeah, I got some ideas for improvement! As for the point you just mentioned, we actually switched to an entirely different parser, so the new version supports nested structs (also anonymous) and even unions :)

**Comment by Hadatko** [comment]
Hello. Unfortunately i don't see progress on these tasks. Did you stopped development?

**Comment by denniskrul** [comment]
Hi there! We got sidetracked for a bit while working on other projects, but development is still ongoing and we expect to be able to release an update around the end of September or start of October.
